### PR TITLE
docs: update docs and license

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
-Our development setup assumes a LINUX/BSD environemnt. Windows users may have to use cygwin.
+Our development setup assumes a LINUX/BSD environemnt.
 
-1. Install Node v8.10.0 or higher.
-2. Install yarn package manager `https://yarnpkg.com/en/docs/install` v1.12 or higher.
-3. Fork the upstream repo `https://github.com/box/box-ui-elements`.
+## Project Setup
+
+1. Install Node v10.0.0 or higher.
+2. Install yarn package manager `https://yarnpkg.com/en/docs/install` v1.10 or higher.
```
Comment (jstoffan) [inline DEVELOPING.md]: `Yarn` is capitalized in their own docs.

File: conf/license.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = `Box UI Element
 
-Copyright 2017 Box, Inc. All rights reserved.
+Copyright 2017-present Box, Inc. All rights reserved.
```
Comment (jstoffan) [inline conf/license.js]: Approved by Legal?

Comment (priyajeet) [reply conf/license.js]: Just following webapp

**Pull request merged**